doc/source/indexing.rst

```diff
@@ -364,7 +369,7 @@ With a DataFrame
    df1
    df1.loc[['a', 'b', 'd'], :]
 
-Accessing via label slices
+Accessing via label slices (also see :ref:`Slicing with labels <indexing.slicing_with_labels>`)
```

Comment:
I wouldn't add this reference on this one (maybe in text some where, but not the section head)